README.md

```diff
@@ -83,6 +83,7 @@ dexcount {
     printAsTree = false
     orderByMethodCount = false
     verbose = false
+    exportAsCSV = false
```

Comment:
Since this is going to be useful primarily for CI environments, it makes more sense to me that we would set this option as an environment variable or a dynamic Gradle property, rather than something hard-coded into the build.

I'm thinking something like `JENKINS_CSV_PATH`, which would contain the path in the workspace of the .csv file.